discovery-provider/src/queries/get_health.py

```diff
@@ -74,6 +74,18 @@ def _get_db_conn_state():
     return conn_state, False
 
 
+# Returns query insights
+def _get_query_insights():
+    query_insights = monitors.get_monitors(
```

Comment:
Yes, I thought this was kind of strange as well but the conn_state response has the same behavior. Should we adjust both of these to return False on errors?

Reply:
oh since this only fetches from redis, this handles errors in the deserialization https://github.com/AudiusProject/audius-protocol/blob/master/discovery-provider/src/monitors/monitors.py#L167-L182